.github/workflows/runE2ETest.yml

```diff
@@ -29,6 +29,16 @@ jobs:
         with:
           path: ./salesforcedx-vscode
           ref: ${{ github.event.ref }}
+      - name: Download extension vsixes
+        run: |
+          mkdir ./extensions
+          pwd
+          gh release download -D ./extensions -p '*.vsix'
```

Comment:
This downloads the latest release?

Reply:
Do we need a follow up PR to be able to target specific builds?

Reply:
> This downloads the latest release?

Yes!

Reply:
> Do we need a follow up PR to be able to target specific builds?

Yes :)